boards/frdm-kw41z/include/board.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2017 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    board_frdm-kw41z Freescale FRDM-KW41Z Board
+ * @ingroup     boards
+ * @brief       Board specific implementations for the FRDM-KW41Z
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the FRDM-KW41Z
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * @brief   LED pin definitions and handlers
+ * @{
+ */
+#define LED0_PIN            GPIO_PIN(PORT_C,  1)
+#define LED0_MASK           (1 << 1)
+#define LED0_ON            (GPIOC->PCOR = LED0_MASK)
+#define LED0_OFF           (GPIOC->PSOR = LED0_MASK)
+#define LED0_TOGGLE        (GPIOC->PTOR = LED0_MASK)
+#define LED1_PIN            GPIO_PIN(PORT_A,  19)
+#define LED1_MASK           (1 << 19)
+#define LED1_ON            (GPIOA->PCOR = LED1_MASK)
+#define LED1_OFF           (GPIOA->PSOR = LED1_MASK)
+#define LED1_TOGGLE        (GPIOA->PTOR = LED1_MASK)
+#define LED2_PIN            GPIO_PIN(PORT_A,  18)
+#define LED2_MASK           (1 << 18)
+#define LED2_ON            (GPIOA->PCOR = LED2_MASK)
+#define LED2_OFF           (GPIOA->PSOR = LED2_MASK)
+#define LED2_TOGGLE        (GPIOA->PTOR = LED2_MASK)
+/** @} */
+
+/**
+ * @brief   xtimer configuration
+ * @{
+ */
+#if 1
+/* LPTMR xtimer configuration */
+/* WIP, Use PIT for now */
```

Comment:
this is not true currently, is it?

It could also be nice, if you would add a small comment, that per default we use the rtt as base for the xtimer, but that one can simply switch to the PIT timer by changing to `#if 0`. Or better: how about we guard this block with something like 
```c
#ifndef FRDM_XTIMER_SOURCE_PIT
...
#else
...
#endif
```
This way one could switch the xtimer configuration via env vars or in the APP makefile